dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -127,6 +131,20 @@
     @UpdateTimestamp
     private Timestamp dbUpdateDate;
 
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(name = "version_input_file_format", joinColumns = @JoinColumn(name = "versionid", referencedColumnName = "id"), inverseJoinColumns = @JoinColumn(name = "fileformatid", referencedColumnName = "id"))
```

Comment:
Since the fileformat table is `fileformat` just use use `version_input_fileformat`.